static/scss/generic/_vertical-divider.scss

```diff
@@ -11,11 +11,11 @@
 ///     <div class="six-col">...</div>
 ///   <div class="six-col last-col">...</div>
 ///   </div>
-@mixin maas-vertical-divider {
+@mixin maas-dividers {
   .vertical-divider {
     position: relative;
 
-    > article {
+    > * {
```

Comment:
You should target a class name here to give you freedom to apply the styling to individual elements.